cmd/commands/root.go

```diff
@@ -0,0 +1,79 @@
+package command
```

Comment:
Could we actually break this out in a separate PR? The first PR could have the makefile and just the most basic command like printing help. Then we can have follow-up PRs to start adding each command.

Reply:
I agree splitting will make the PR easier to review and also for me to make review changes. Here is the #3 PR.

Reply:
Let's get the pr #3 ready and merge and then we'll come back to this PR and any other PR for reviews and changes.